### ISSUE TITLE
`model.predict_classes()` is deprecated

### ISSUE THREAD

**Issue by gen-li** [appeler/ethnicolr#44]
Hi, 

I got the warning below when you run your module.

 455: UserWarning: `model.predict_classes()` is deprecated and will be removed after 2021-01-01. Please use instead:* `np.argmax(model.predict(x), axis=-1)`,   if your model does multi-class classification   (e.g. if it uses a `softmax` last-layer activation).* `(model.predict(x) > 0.5).astype("int32")`,   if your model does binary classification   (e.g. if it uses a `sigmoid` last-layer activation).

  warnings.warn('`model.predict_classes()` is deprecated and '
2021-07-08 19:20:01.005812: I tensorflow/compiler/mlir/mlir_graph_optimization_pass.cc:176] None of the MLIR Optimization Passes are enabled (registered 2)

**Comment by suriyan** [comment]
Thanks for reporting the issue. Fixed in the latest version.
